README.md

```diff
@@ -1,2 +1 @@
-# projectreview
-Microverse Trials Project Review
```

Comment:
- [ ] I believe it would be better if you add more info to the README file. :ok_hand:. It's always nice to have a good idea what the project is about, and even how it looks like, from the README file. This way your project will look way more professional! :thumbsup: